cg/models/taxprofiler/taxprofiler.py

```diff
@@ -1,9 +1,20 @@
 from pathlib import Path
 
-from pydantic.v1 import Field
+from pydantic.v1 import Field, BaseModel
+from typing import Optional
 
 from cg.constants.sequencing import SequencingPlatform
 from cg.models.nf_analysis import NextflowSampleSheetEntry, PipelineParameters
+from cg.models.analysis import AnalysisModel
+
+
+class TaxprofilerQCMetrics(BaseModel):
+    """Taxprofiler QC metrics."""
+
+    after_filtering_total_reads: Optional[float]
+    reads_mapped: Optional[float]
+    before_filtering_total_reads: Optional[float]
+    paired_aligned_none: Optional[float]
```

Comment:
The model might need adjustment to reflect pydantic v2 usage Optional values that are not always parsed require to be initialised as None
```suggestion
    after_filtering_total_reads: Optional[float] = None
    reads_mapped: Optional[float] = None
    before_filtering_total_reads: Optional[float] = None
    paired_aligned_none: Optional[float] = None
```